mm2src/coins/lp_coins.rs

```diff
@@ -995,6 +1021,15 @@ pub enum CoinProtocol {
         platform: String,
         contract_address: String,
     },
+    SLPTOKEN {
+        platform: String,
+        token_id: H256Json,
+        decimals: u8,
+        required_confirmations: Option<u64>,
```

Comment:
SLP tokens can have from 0 to at least 8 decimals. It's possible to request it during coin initialization, but it might be an unnecessary overhead. 
As per `required_confirmations`, I preferred to leave the possibility to set a different number from the base coin. E.g. one may want to have a bigger number for BCH and a lower one for a specific token.

Reply:
Got it. I'm afraid the number of fields within the object may increase in the future, so we need to be careful with adding new fields.